docs/zkapps/how-zkapps-work.mdx

```diff
@@ -24,153 +24,139 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
-Both **private inputs** and **public inputs** represent data that must be provided to the prover function when it runs in the user’s web browser.
+Both private inputs and public inputs represent data that must be provided to the prover function when it runs in the user's web browser.
 
-**Private inputs** are not needed again after that point. But **public inputs** must be also provided to the verifier function when it runs on the Mina network, and as such should never be used for data that you want to remain private.
+Private inputs are not required again. Because public inputs must also be provided to the verifier function when it runs on the Mina network, public inputs are not used for data that you want to remain private.
 
-The **verifier function** is the function that validates whether a zero-knowledge proof successfully passes all the constraints defined in the prover function. This _always_ runs quickly and efficiently _irrespective of the prover function’s complexity_.
+The verifier function validates whether a zero knowledge proof successfully passes all the constraints defined in the prover function. The verifier function _always_ runs quickly and efficiently, irrespective of the prover function's complexity.
 
 Within the Mina network, Mina acts as the verifier and runs the verifier function. 
 
 <figure>
-  <img src="/img/5_zkApps_Verifier_Function.jpg" width="90%" />
-  <figcaption>The verifier function is the function that validates whether a zero-knowledge proof successfully passes all the constraints defined in the prover function.</figcaption>
+  <img src="/img/5_zkApps_Verifier_Function.jpg" width="90%" alt="Diagram showing verifier function validation" />
 </figure>
 
-### Prover Function & Verification Key
+## Prover Function and Verification Key
 
-The zkApp developer writes smart contracts and then uses the `npm run build` command. The build process outputs `smart_contract.js`. From this, you can:
+After you write your smart contract, run the `npm run build` command. The build process compiles the TypeScript code into JavaScript and outputs the `smart_contract.js` file. 
 
-- Run your prover function to run your smart contract
-- Generate a verification key to deploy your smart contract
+From this file, you can:
 
-The **prover function** runs in a user’s web browser.
+- Run a prover function to run your smart contract
+- Generate a verification key to deploy your smart contract
 
-The **verification key** lives on chain for a given zkApp account and is used by the Mina network to verify that a zero-knowledge proof has met all constraints defined in the prover. A verification key is required in order to create a zkApp account. 
+While the prover function runs in a user's web browser, the verification key lives on-chain for a given zkApp account and is used by the Mina network to verify that a zero knowledge proof has met all constraints defined in the prover. A verification key is required to create a zkApp account.
```

Comment:
You can also use the verifier function (or verification key) to verify proofs off-chain, but yes the main usage for the verifier function is the Mina network